packages/components/avatar/src/avatar.tsx

```diff
@@ -10,7 +10,7 @@ import {
 import { useImage } from "@yamada-ui/image"
 import { createContext, cx, dataAttr, handlerAll } from "@yamada-ui/utils"
 import { cloneElement, useState } from "react"
-import { AvatarIcon } from "./avatar-icon"
+import { UserRoundIcon } from "../../lucide/src"
```

Comment:
I think you need to fix this line.
```suggestion
import { UserRoundIcon } from "@yamada-ui/lucide"
```

Reply:
FIxed.